src/shared/models/fiat/__tests__/mock/fiat.entity.mock.ts

```diff
@@ -0,0 +1,14 @@
+import { Fiat } from '../../fiat.entity';
+
+const defaultFiat: Partial<Fiat> = {
+  name: 'EUR',
+  enable: true,
+};
+
+export function createDefaultFiat(): Fiat {
+  return createCustomFiat({});
+}
+
+export function createCustomFiat(customValues: Partial<Fiat>): Fiat {
+  return { ...new Fiat(), ...defaultFiat, ...customValues };
```

Comment:
```ts
const fiat = {...new Fiat()};
fiat instanceof Fiat // false
```
Perhaps better use `Object.assign(new Fiat(), { ...defaultFiat, ...customValues })` (for all mocks)?

Reply:
oh. wasn't aware of this. Then yes, we should definitely change it for all mocks